mixer/pkg/server/server.go

```diff
@@ -253,6 +253,9 @@ func (s *Server) Close() error {
 		_ = os.RemoveAll(s.configDir)
 	}
 
+	// final attempt to purge buffered logs
+	log.Sync()
```

Comment:
Is this a bug fix or a side-effect of this effort?

Reply:
Just to tie this back to the same thread... from @geeknoid:

>A preemptive bug fix. I just noticed I forgot a flush here for good measure.

SGTM - thanks!